jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -130,20 +138,24 @@ public void preConfigure(WebAppContext context) throws Exception
                 else
                 {
                     if (LOG.isDebugEnabled())
-                        LOG.debug("No quickstart xml file, starting webapp {} normally", context);
-                    super.preConfigure(context);
```

Comment:
Leave `super` call in here, removing it hasn't got anything to do with the issue and I think minimal changes are best.

Reply:
Done.